### PR TITLE
mgr/rook: Fix creation of bluestore OSDs

### PR DESCRIPTION
**Pull request by sebastian-philipp**
See https://rook.io/docs/rook/v0.9/ceph-cluster-crd.html
    Storage Configuration: Specific devices

Fixes http://tracker.ceph.com/issues/39062

Signed-off-by: Sebastian Wagner <sebastian.wagner@suse.com>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

**Comment by sebastian-philipp** [comment]
according to @travisn , replacing `storeConfig` with `config` is correct

**Comment by sebastian-philipp** [comment]
Relates to 

https://github.com/rook/rook/blob/0e7dbe408622d4343238484f34fa50c42de7ca7d/cluster/examples/kubernetes/ceph/operator.yaml#L64-L66

File: src/pybind/mgr/rook/rook_cluster.py [hunk 1]
```diff
@@ -398,7 +398,7 @@ def add_osds(self, drive_group, all_hosts):
 
         if drive_group.hosts(all_hosts)[0] not in [n['name'] for n in current_nodes]:
             pd = { "name": drive_group.hosts(all_hosts)[0],
-                   "storeConfig": { "storeType": drive_group.objectstore }}
+                   "config": { "storeType": drive_group.objectstore }}
```
Comment (jtlayton) [inline src/pybind/mgr/rook/rook_cluster.py]: Could you also fix up the comment that is about 20 lines above?

Comment (sebastian-philipp) [reply src/pybind/mgr/rook/rook_cluster.py]: fixed

**Pull request merged**